ts/components/conversation/TimelineMessage.tsx

```diff
@@ -23,7 +23,7 @@ import type {
   PropsData as MessagePropsData,
   PropsHousekeeping,
 } from './Message';
-import type { PushPanelForConversationActionType } from '../../state/ducks/conversations';
+import { type PushPanelForConversationActionType } from '../../state/ducks/conversations';
```

Comment:
I think you want the `type` to be outside the braces, in fact. That's generally how we do things.

Reply:
I reverted the change, but shouldn't `npm run format` take care of this.
I cannot remember that I did this change intentionally and had hoped that Prettier will include a rule for these imports.